RobotCode/src/main/java/frc/lib/modules/swervedrive/Commands/DriveDistanceAuto.java

```diff
@@ -7,58 +7,111 @@
 import frc.lib.modules.swervedrive.SwerveDriveSubsystem;
 
 /**
- * @deprecated Use PathPlanner instead
+ * Includes both movement and rotation. Rotation is in degrees for this command.
  */
-@Deprecated
 public class DriveDistanceAuto extends Command
 {
-	DriveDistanceAuto driveDistanceAuto;
-	private double distance;
 
-	SwerveDriveSubsystem s_Swerve;
+	private final SwerveDriveSubsystem Swerve;
```

Comment:
Shouldn't this be capitalized if this is final, or is that only if it's a constant (static)

Reply:
Final objects use upper camelcase. It would be SWERVE if it was a primitive.